### DISCUSSION TITLE
Why rh Voice does not support Vietnamese language?

### DISCUSSION THREAD

**Discussion by NVCDevelopmentTeam** [RHVoice/RHVoice#761] [General]
I have discussed with the rh voice development department about the Vietnamese native language issue, but as of now, rh voice is not supported in the current language. So I think there should be a specific discussion on this issue. In your opinion, how to integrate this language into rh voice?

**Comment by NVCDevelopmentTeam** [comment]
I think, need to support this language when everything is ready.

**Reply by louderpages** [reply]
Coming soon:  https://louderpages.org/atscale-global-partnership-four-new-languages/

**Comment by NVCDevelopmentTeam** [comment]
I will wait for the demo version and its completed full version. However, it is necessary to consult with users after launching the official version to ensure that it does not encounter any problems. Thank.
